client-core/src/signer/key_pair_signer.rs

```diff
@@ -0,0 +1,81 @@
+//! A signer that sign message using the provided key pair
```

Comment:
could this made `1-of-N` easily (or can leave a todo comment), or rename it to reflect that it's 1-of-1?

Reply:
Yes, I think so. I can let the user register 1-of-n addresses, then we can utilize the information to sign inputs. This will also make `MultiSignUnlock` in `SignCondition` enum possible.

I will now leave it as TODO first and add it when l I add support of MultiSig support to JS lib.